Fix the generation of the 64-bit /usr/lib/64/pkgconfig/autoopts.pc
for Solaris based on changes to .../autoopts/mk-autoopts-pc.in already
present upstream.

--- autogen-5.16.2/autoopts/mk-autoopts-pc.in.orig	2016-01-19 16:17:03.319088249 -0800
+++ autogen-5.16.2/autoopts/mk-autoopts-pc.in	2016-01-19 16:17:23.599671581 -0800
@@ -31,7 +31,7 @@
       cflags="-I${includedir}"
 test 'X@ENABLE_STATIC@' = Xno && static_libs=''
 case "${libdir}" in
-/lib | /lib64 | /usr/lib | /usr/lib64 )
+/lib | /lib64 | /usr/lib | /usr/lib64 | /usr/lib/* )
     ldopts=''
     ldflags=-lopts
     ;;
